README.md

```diff
@@ -86,4 +293,110 @@
+
+* IP ranges, subnet mask, and a reliable L3 (default) gateway for each VLAN are provided.
+
+* Jumbo Frames (MTU 9000) are recommended on all VLANs. At a minimum, MTU of 1600 is required on the NSX Host Overlay VLAN and must be enabled end to end through your environment.
+
+* VLANs for management, vMotion, vSAN and NSX Host Overlay networks are created and tagged to all host ports. Each VLAN is 802.1q tagged.
+
+* Management IP is VLAN backed and configured on the host. vMotion & vSAN IP ranges are configured during the bring-up process.
+
+#### Physical Hardware and ESXi Host
+
+* All servers are vSAN compliant and certified on the VMware Hardware Compatibility Guide, including but not limited to BIOS, HBA, SSD, HDD, etc.
+
+* Identical hardware (CPU, Memory, NICs, SSD/HDD, etc.) within the management cluster is highly recommended. Refer to vSAN documentation for minimal configuration.
+
+* One physical NIC is configured and connected to the vSphere Standard switch. The second physical NIC is not configured.
+
+* ESXi is freshly installed on each host. The ESXi version matches the build listed in the Cloud Foundation Bill of Materials.
+
+* All hosts are configured with a central time server (NTP). NTP service policy set to 'Start and stop with host'.
+
+* Each ESXi host is running a non-expired license - initial evaluation license is accepted. The bring-up process will configure the permanent license provided.
+
+#### Supporting Infrastructure
+
+* All hosts are configured with a DNS server for name resolution.
+
+## Custom root password
+
+### Generating custom root password
+
+To generate a password hash of your desired ESXi root password run the 'mkpasswd' command on a Linux system with the 'whois' package installed as follows
+
+```shell
+mkpasswd --method=SHA-512
+```
+
+You'll be prompted to enter the desired password sting you wish to hash, then press enter.
+
+![Alt text](assets/9-mkpasswd_example.png "mkpasswd Example")
+
+The output will be the string you need to use in the esxi_password variable near the end of the terraform.tfvars.example file
+
+## Terraform Deployment Workflow
+
+### Preparation
+
+* Download the vcf-ems-deployment-parameter_X.X.X spreadsheet for the VCF version you're deploying from VMware and fill it out for your environment.
+* Copy the terraform.tfvars.example file to terraform.tfvars and fill it with the same values you used in the vcf-ems-deployment-parameter_X.X.X spreadsheet.
+
+### Deployment
+
+* Deploy this Terraform module
+* RDP to the management host
+  * Username: `SYSTEM\Admin`
+  * Password: provided in the terraform output `terraform output -raw management_password`.
+* Download the Cloudbuilder OVA from VMware
+* Log in to one of the ESXi hosts
+  * <https://sfo01-m01-esx01.sfo.rainpole.io>
+* Deploy Cloudbuilder OVA on ESXi
+* Login to cloudbuilder at <https://172.16.9.3> with admin/<the password you chose>
```

Comment:
are the IP or password known in Terraform?

Reply:
The IP is, currently the password only exists in the spreadsheet. The _hash_ of that password is in terraform.

Reply:
I've removed the IP and password stuff here, and just said to use what you used when you deploy the OVA.